docs/src/pages/company/software-engineer-intern/software-engineer-intern.md

```diff
@@ -0,0 +1,78 @@
+- **You'll be part of an active, open, friendly community** of developers that are really excited about building awesome applications.
+- **Your work will be visible** as being part of one of the most popular open source project.
+
+### The worst parts of this job
+
+- **Shifting context.** You will necessarily have to shift context and dive into a different feature before the current one is done. It may even be in an area of the code base you're unfamiliar with or don't have a ton of understanding about. It's fun, rewarding work, but it can be very challenging.
+- **We move quickly, but don't sacrifice quality**. We ship early, often, and quickly. You may not be initially comfortable with the cadence with which we ship high-quality features and improvements to end-users. By doing so, we sacrifice on solving each problem 100% in exchange for fast feedback. Solving 50-70% of the issue with quality should be enough for any given iteration. Our users quickly tell us when we haven't pushed a solution far enough.
+- **Material-UI is a large codebase**. You may bang your head against the wall at times, and then write tests to assist future you 😌.
+  The work you will be doing is somewhat unique and idiosyncratic. You probably have not had a similar role before.
+- **In open-source, you're faced with a nonstop stream of bug reports and support requests**. That means you need to develop an intuition for when to ignore something, and when to dig in further.
+
+## About you
+
+We're looking for someone with strong front-end skills or someone who has a passion and is eager to learn more in this area. More important than specific technical skills though is that you're a strong problem solver who loves to learn. Details matter to you.
+
+### Skills you should have
+
+- **Experience in the modern JavaScript ecosystem**. Material-UI is built on the shoulders of giants, making use of technologies such as ES2015+, TypeScript, Node.js, React, Next.js, webpack, and Babel.
+- **Strong written and verbal communication skills**. As part of the team, you'll interface both directly and indirectly with community members and enterprise customers, and contribute to user documentation. Clear communication is fundamental in creating intuitive and compelling resources.
+- **Eagerness to dive into complex problems**. You should be able to quickly assess, understand, and iterate upon aspects of our codebase.
+- **Ready and willing to ask and answer questions**. If you're comfortable saying you're unsure, asking for help; but equally reaching out to assist others, you'll be an incredible addition to our team. We thrive because of continuous learning. First-time mistakes should be celebrated, not blamed.
+- **Past work with frontend infrastructure**. Perheps you've had a scholl project using React components. It would be great if you could address this in your cover letter!
```

Comment:
```suggestion
- **Past work with frontend infrastructure**. Perhaps you've had a school project using React components. It would be great if you could address this in your cover letter!
```